sql/connect/server/src/test/scala/org/apache/spark/sql/connect/pipelines/SparkDeclarativePipelinesServerSuite.scala

```diff
@@ -486,4 +486,138 @@ class SparkDeclarativePipelinesServerSuite
       assert(graphsAfter.isEmpty, "Graph should be removed after drop")
     }
   }
+
+  gridTest("DefineDataset returns resolved data name for default catalog/schema")(
+    Seq(
+      ("TEMPORARY_VIEW", DatasetType.TEMPORARY_VIEW, "tv"),
+      ("TABLE", DatasetType.TABLE, "tb"),
+      ("MV", DatasetType.MATERIALIZED_VIEW, "mv")
+    )
+  ) { case (_, datasetType, datasetName) =>
```

Comment:
nit and optional, you can use namedGridTest then if you wanna give each test a bane